### PR TITLE
Reviewbranch

### PR DESCRIPTION
**Pull request by Chinoms**
Added New Class - ```homescreen```
   The class handles everything on the homescreen.
   Only current method handles the display of featured categories on the homescreen.
   Added placeholder image for testing purposes

### CODE REVIEW

File: modules/homeScreen.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+require_once("../classes/homescreen.php");
```
Comment (basilcea) [inline modules/homeScreen.php:2]: why require once and not require

**Review by basilcea — COMMENTED** [review]
Nice Job Nonso. I Left a comment for you

File: modules/homeScreen.php (hunk 1 above)
Comment (Chinoms) [reply modules/homeScreen.php:2]: Thanks, @basilcea. ```require_once``` is an alias for ```require```, except that ```require_once``` checks if the file has been previously included and skips if it.

Comment (basilcea) [reply modules/homeScreen.php:2]: got it

**Pull request merged**